extensions/rest-client/deployment/src/main/java/io/quarkus/restclient/deployment/RestClientProcessor.java

```diff
@@ -83,6 +83,9 @@
     private static final DotName REGISTER_PROVIDER = DotName.createSimple(RegisterProvider.class.getName());
     private static final DotName REGISTER_PROVIDERS = DotName.createSimple(RegisterProviders.class.getName());
 
+    private static final DotName CLIENT_REQUEST_FILTER = DotName.createSimple(ClientRequestFilter.class.getName());
```

Comment:
Does it make sense to support `WriteInterceptor` and `ReadInterceptor` as well ? Though it can be done as part of another PR for sure...

Reply:
hi @sberyozkin, yes I think that would also make sense. Since those interfaces are part of the `javax.ws.rs.ext` package, it would probably be best to register them in the core "resteasy-common" module. I'll open a follow-up PR with the additions and tests in the next day or so

Reply:
Hi @aguibert sorry for a delay; thanks, that would be great...Cheers